meta-ivi/recipes-extended/dlt-daemon/dlt-daemon_2.18.1.bb

```diff
@@ -12,26 +12,25 @@ SECTION = "console/utils"
 LICENSE = "MPLv2"
 LIC_FILES_CHKSUM = "file://LICENSE;md5=8184208060df880fe3137b93eb88aeea"
 
-PR = "r2"
-
 DEPENDS = "dbus zlib gzip-native"
 
-SRCREV = "bdde2374efd567f27d7a4ec70911b34cda061d27"
-SRC_URI = "git://github.com/GENIVI/${BPN}.git;protocol=https \
+SRC_URI = "https://github.com/GENIVI/dlt-daemon/archive/v${PV}.tar.gz \
```

Comment:
I am OK with this.  It hurts from one perspective because I think it would make some sense to use git for all sources all the time.  But the way things are done in OE/yocto I have to admit that tarballs are more straight forward (git repos use weird storage tricks in the downloads cache, and unexpected checks to see if a tag has moved messes up BB_NO_NETWORK setups).  Tar is just easier to handle in OE/yocto world.

We have no particular documented policy for meta-ivi as far as I know.